include/seqan/index/index_qgram.h

```diff
@@ -275,15 +275,9 @@ class Index<TText_, IndexQGram<TShapeSpec, TSpec> > {
     cargo(other.cargo),
     stepSize(1) {}
 
-    Index(Index const &other):
```

Comment:
do you know why there is another copy constructor that takes the index as non const reference?

Reply:
I don't really know, but compiling fails if you don't have the non const version...

Reply:
Wow...